packages/auth/src/platform_cordova/popup_redirect/events.test.ts

```diff
@@ -73,28 +70,28 @@ describe('platform_cordova/popup_redirect/events', () => {
 
   describe('_savePartialEvent', () => {
     it('sets the event', async () => {
+      const spy = sinon.spy(Storage.prototype, 'setItem');
       const event = _generateNewEvent(auth, AuthEventType.REAUTH_VIA_REDIRECT);
       await _savePartialEvent(auth, event);
-      expect(storageStub.setItem).to.have.been.calledWith(
-        'firebase:authEvent:test-api-key:test-app',
-        JSON.stringify(event)
-      );
+      spy.calledWith('firebase:authEvent:test-api-key:test-app',
```

Comment:
Does this not need to be wrapped in expect()?

Reply:
Yes, adding it now